cpp/src/quantiles/tdigest/tdigest_aggregation.cu

```diff
@@ -366,8 +366,8 @@ std::unique_ptr<scalar> to_tdigest_scalar(std::unique_ptr<column>&& tdigest,
  * @param group_cluster_wl    Output.  The set of cluster weight limits for each group.
  * @param group_num_clusters  Output.  The number of output clusters for each input group.
  * @param group_cluster_offsets  Offsets per-group to the start of it's clusters
- * @param has_nulls Whether or not the input contains nulls
- *
+ * @param may_have_empty_clusters Whether or not there could be empty clusters. False if there
+ * should be no empty clusters.
```

Comment:
```suggestion
 * @param may_have_empty_clusters Whether or not there could be empty clusters.
```

Reply:
```
 * @param may_have_empty_clusters Whether or not there could be empty clusters. It should be
 * set to false only when there is no empty cluster.
```

Thanks @mhaseeb123 for suggestion. Though, I'd like to keep the second part of the description to clarify when it should be set to false. I have rephrased it to the above to make it clearer. Let me know what you think.